src/effects/effectxmlelements.h

```diff
@@ -6,9 +6,6 @@ const int kXmlSchemaVersion = 0;
 const QString Root("MixxxEffects");
 const QString SchemaVersion("SchemaVersion");
 
-const QString Rack("Rack");
```

Comment:
This is still required for backward compatibility loading effects.xml files from older versions of Mixxx.